### PR TITLE
ticket service added

### PR DESCRIPTION
**Pull request by vvviin**

### CODE REVIEW

File: src/main/java/com/ty/event/event_management/controller/TicketControlller.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import com.ty.event.event_management.dto.Ticket;
+import com.ty.event.event_management.service.TicketService;
+import com.ty.event.event_management.util.ResponseStructure;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+@RestController
+@RequestMapping("Ticket")
```
Comment (sagarb2722) [inline src/main/java/com/ty/event/event_management/controller/TicketControlller.java:27]: checck once

File: src/main/java/com/ty/event/event_management/controller/TicketControlller.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	}
+
+	@PutMapping
+	public ResponseEntity<ResponseStructure<Ticket>> updateTicket(@RequestBody Ticket ticket, @RequestParam int id) {
+
+		return ticketService.updateTicketById(ticket, id);
+	}
+
+	@GetMapping
+	public ResponseEntity<ResponseStructure<Ticket>> grtTicketByid(@RequestParam int id) {
```
Comment (GSTharun) [inline src/main/java/com/ty/event/event_management/controller/TicketControlller.java:46]: change get spelling

File: src/main/java/com/ty/event/event_management/controller/TicketControlller.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+		return ticketService.deleteTicketById(id);
```
Comment (GSTharun) [inline src/main/java/com/ty/event/event_management/controller/TicketControlller.java:48]: change update method

File: src/main/java/com/ty/event/event_management/controller/TicketControlller.java (hunk 1 above)
Comment (GSTharun) [inline src/main/java/com/ty/event/event_management/controller/TicketControlller.java:27]: make  the url smaller case

**Review by GSTharun — CHANGES_REQUESTED** [review]

File: src/main/java/com/ty/event/event_management/controller/TicketControlller.java (hunk 3 above)
Comment (sahanajavadevelop) [inline src/main/java/com/ty/event/event_management/controller/TicketControlller.java:48]: change the method name

**Review by sahanajavadevelop — CHANGES_REQUESTED** [review]

File: src/main/java/com/ty/event/event_management/controller/TicketControlller.java (hunk 1 above)
Comment (vvviin) [reply src/main/java/com/ty/event/event_management/controller/TicketControlller.java:27]: done

**Review by GSTharun — APPROVED** [review]

**Review by sahanajavadevelop — APPROVED** [review]

**Pull request merged**